tests/unit/test_agent_skill.py

```diff
@@ -1588,3 +1589,63 @@ def test_lint_file_fail_non_python(tmp_path, capsys):
             'DO NOT re-run the same failed edit command. Running it again will lead to the same error.\n'
         )
         assert result.split('\n') == expected.split('\n')
+
+
+def test_lint_file_fail_typescript(tmp_path, capsys):
+    with patch.dict(os.environ, {'ENABLE_AUTO_LINT': 'True'}):
+        current_line = 1
+        file_path = tmp_path / 'test.ts'
+        file_path.write_text('')  # Create an empty TypeScript file
+
+        open_file(str(file_path), current_line)
+        insert_content_at_line(
+            str(file_path),
+            1,
+            "function greet(name: string) {\n    console.log('Hello, ' + name)\n",
+        )
+        result = capsys.readouterr().out
+        assert result is not None
+
+        # Use a regex to match the file path in the error message
+        error_line_pattern = r'.*test\.ts\(\d+,\d+\): error TS\d+: .*'
```

Comment:
(this is just my personal opinion) can we use actual messages rather than regex in tests? Regex are hard to comprehend and maintain

Reply:
The reason for using the regex was, that the typescript compiler could return different line numbers in WSL compared to CI and its path was a relative path with lots of `../`.
I changed it back to a "normal" format without regex, will see how the test works in CI now.